cloudify_docker_image/make_cfy_manager_image.sh

```diff
@@ -11,17 +11,27 @@ CFY_RPM="cloudify-manager-install.rpm"
 CONTAINER_NAME="cfy_manager"
 BASE_IMAGE="cfy_manager_base"
 IMAGE_PUB_NAME="docker_cfy_manager"
+declare -a IMAGE_TAGS=( "latest" "centos7_v1.0_manager_v4.3" )
 DOCKER_RUN_FLAGS="--name ${CONTAINER_NAME} -d -v /sys/fs/cgroup:/sys/fs/cgroup:ro --tmpfs /run
  --tmpfs /run/lock --security-opt seccomp:unconfined --cap-add SYS_ADMIN"
 MANAGER_CONFIG_LOCATION="/opt/cloudify"
+DOCKER_ID_USER=
```

Comment:
Let's try to get those from env vars instead of leaving it blank. If we'll want to use this script in automation, we won't have to edit it, but rather could setup env vars